nobots.module

```diff
@@ -8,7 +8,9 @@
  * Implements hook_init().
  */
 function nobots_init() {
-  drupal_add_http_header("X-Robots-Tag", "noindex,nofollow,noarchive");
+  if (variable_get('nobots') == TRUE) {
```

Comment:
Oh yeah, I was thinking about that and forgot to implement it that way. I'll fix.